app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -116,6 +118,13 @@ class ProfileEditActivityTest {
   @get:Rule
   val oppiaTestRule = OppiaTestRule()
 
+  @get:Rule
+  var activityTestRule: ActivityTestRule<ProfileListActivity> = ActivityTestRule(
```

Comment:
We shouldn't be using ActivityTestRule anymore (it's deprecated), plus we shouldn't be using another test suite's activity.

Reply:
I have removed this.